src/darwin/Framework/CHIPTests/MTRCertificateTests.m

```diff
@@ -277,7 +293,7 @@ - (void)testGenerateOperationalCertNoIntermediateWithValidityPeriod
 
     __auto_type * operationalCert = [MTRCertificates createOperationalCertificate:rootKeys
                                                                signingCertificate:rootCert
-                                                             operationalPublicKey:operationalKeys.publicKey
+                                                             operationalPublicKey:operationalPublicKey
```

Comment:
Why not copy autorelease this? Makes for much easier code...

Reply:
(discussing)